data/json/recipes/recipe_armor.json

```diff
@@ -9074,5 +9074,80 @@
         [["duct_tape", 35], ["superglue", 5]],
         [["jerrycan", 1]]
     ]
-  }
+  },{
+  "type" : "recipe",
+  "result": "wetsuit",
+  "category": "CC_ARMOR",
+  "subcategory": "CSC_ARMOR_SUIT",
+  "skill_used": "tailor",
+  "skills_required" : [ "tailor", 4 ],
+  "difficulty": 4,
+  "time": 8000,
+  "reversible": false,
+  "autolearn": false,
+  "book_learn": [[ "textbook_tailor", 4 ] , [ "manual_tailor", 5 ] , ["tailor_portfolio", 4]],
+  "tools": [
+    [
+      [ "sewing_kit", 50 ]
```

Comment:
Sewing kit isn't enough to mend the sheets into a single suit, there needs to be some sort of heat source here: either soldering iron/welder or hotplate.